apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/enrollment/EnrollmentServiceImpl.java

```diff
@@ -65,6 +69,12 @@ public Enrollment signUp(Long eventId) {
         return enrollmentRepository.findByUser(authenticationService.getAuthenticatedUser());
     }
 
+    @Override
+    public byte[] generateQRCode(Long enrollmentId) {
+        return qrcodeService.generateQRCode(
+            String.format(confirmURLScheme, enrollmentId), 640, 640);
```

Comment:
I would make those magic numbers into `static private final` variables as these are constants

Reply:
@gajus123 please, make the change and let's merge the PR.